src/ubuntu/24.04/sdk/amd64/Dockerfile

```
@@ -0,0 +1,61 @@
+FROM mcr.microsoft.com/dotnet/sdk:10.0-preview
```

The tag should be explicit about which distro version it's expecting. 

```suggestion
FROM mcr.microsoft.com/dotnet/sdk:10.0-preview-noble
```